operators/src/plot/histogram.rs

```diff
@@ -141,6 +146,10 @@ impl PlotOperator for Histogram {
             }
         })
     }
+
+    fn span(&self) -> CreateSpan {
+        || span!(Level::TRACE, Histogram::TYPE_NAME)
+    }
```

Comment:
Maybe we can at least use a macro for this `fn`. This way, we can easily change the behavior for all operators.

Reply:
Would be nice, but we could have a normal macro instead here.

Reply:
de3d837b15aa93ceeaa690437b8747d2d5457067